### ISSUE TITLE
wrap query: queries repositories for packages

### ISSUE THREAD

**Issue by serialseb** [OpenWrap/openwrap#55]
Can do a query at the command-line to search for packages.

```
query-wrap [-query] searchstring [-project|-system|-remote [name]]
```

**Comment by serialseb** [comment]
Change of plan, this was added to the list-wrap instead, why have more commands?

**Comment by serialseb** [comment]
Adds a -query parameter to list-wrap. Closed by 72a9d6324e266668cae44fd7d6070f348d8b1d3e
